src/RcppExports.cpp

```diff
@@ -380,7 +380,7 @@ BEGIN_RCPP
 END_RCPP
 }
 
-RcppExport SEXP run_testthat_tests(SEXP);
+RcppExport SEXP run_testthat_tests(void *);
```

Comment:
Any idea about why this changed? Noticed the same on Friday. Just curious.

Reply:
No idea, but presumably due to changes in either testthat or Rcpp, although I did not find anything that could explain it by browsing through the latest changes in those packages.